env/WCOSS2.env

```diff
@@ -86,40 +91,53 @@ elif [ $step = "gldas" ]; then
 
 elif [ $step = "eobs" ]; then
 
+    export OMP_PLACES=cores
+    export OMP_NUM_THREADS=$nth_eobs
+    export OMP_STACKSIZE=1G
+    export FI_OFI_RXM_SAR_LIMIT=3145728
+
     nth_max=$(($npe_node_max / $npe_node_eobs))
 
     export NTHREADS_GSI=${nth_eobs:-$nth_max}
     [[ $NTHREADS_GSI -gt $nth_max ]] && export NTHREADS_GSI=$nth_max
-    export APRUN_GSI="$launcher -n ${npe_gsi:-${npe_eobs:-$PBS_NP}} -ppn $npe_node_eobs"
+    export APRUN_GSI="$launcher -n ${npe_gsi:-${npe_eobs:-$PBS_NP}} -ppn $npe_node_eobs --cpu-bind core --depth $nth_eobs"
 
     export CFP_MP=${CFP_MP:-"NO"}
     export USE_CFP=${USE_CFP:-"YES"}
     export APRUNCFP="$launcher -np \$ncmd $mpmd"
 
 elif [ $step = "eupd" ]; then
 
+    export OMP_PLACES=cores
+    export OMP_NUM_THREADS=$nth_eupd
```

Comment:
Why is this being set here?  This should be set in the ex-script immediately prior to the execution.
Also, what if `nth_eupd` is not the correct one?  `NTHREADS_ENKF` is the correct one that is being supplied.

Reply:
I have corrected the NTHREADS settings (other comment) and removed the unneeded OMP_NUM_THREADS line.

Is there a question about OMP_PLACES?

Reply:
No question about `OMP_PLACES`.